FWCore/Framework/interface/Run.h

```diff
@@ -129,6 +129,8 @@ namespace edm {
 
     Provenance getProvenance(BranchID const& theID) const;
 
+    StableProvenance getStableProvenance(BranchID const& theID) const;
```

Comment:
Should this be 
```suggestion
    StableProvenance const& getStableProvenance(BranchID const& theID) const;
```
as in `Event.h`?

Reply:
Thanks for the catch. Done.